pulsar-sql/pom.xml

```diff
@@ -32,7 +32,10 @@
     <name>Pulsar SQL :: Parent</name>
 
     <properties>
-        <jackson.version>2.9.7</jackson.version>
+        <jackson.version>2.8.11</jackson.version>
+        <!--fix Security Vulnerabilities-->
+        <!--https://www.cvedetails.com/vulnerability-list/vendor_id-15866/product_id-42991/Fasterxml-Jackson-databind.html-->
+        <jackson.databind.version>2.8.11.1</jackson.databind.version>
```

Comment:
why can't we use `jackson.version` for all jackson dependencies ? I don't think there will be any issue because `2.8.11.1` has only data-bind change on top of `2.8.11`. So, we might not need two separate versions.

Reply:
@rdhabalia there is only 2.8.11.1 for databind, there isn't that version for other jackson dependencies that aren't effect by this vulnerability.

Reply:
hmm.. I see.

Reply:
@rdhabalia This is just for presto, the rest of our modules are still with 2.9.x